src/views/Teams/index.jsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import BugzillaComponents from '../../components/BugzillaComponents';
+
+const Teams = props => (
+  <React.Fragment>
+    <BugzillaComponents {...props} />
```

Comment:
This takes a couple of seconds to render. You can show a spinner for better UX. Also, I would probably have the "If you want to change ..." text above this component.

Reply:
I've moved the message above:
<img width="550" alt="image" src="https://user-images.githubusercontent.com/44410/56136208-563f5100-5f60-11e9-9b63-bbfadbfe8f75.png">